src/components/OrganizationCard/OrganizationCard.test.tsx

```diff
@@ -6,6 +6,7 @@ describe('Testing the LoginPage', () => {
   test('should show the text on the website', () => {
     render(
       <OrganizationCard
+        id="624135624625"
```

Comment:
Please generate a random value for this field.
When writing tests, it's essential to test using random data to ensure that your test doesn't only pass for one scenario